README.md

```diff
@@ -25,9 +25,11 @@ The bindings were generated with the following procedure:
 - Checkout the required version
 - Translate C headers to D modules:
 ```sh
-find src/libsodium/include/sodium -depth 1 -name "*.h" | xargs -I F $DSTEP -Isrc/libsodium/include/sodium/ F -o $LIBSODIUMD/source/libsodium/$(basename F | cut -d'.' -f1).d
+find "$LIPSODIUM_REPO/src/libsodium/include/sodium" -maxdepth 1 -xtype f -name '*.h' \
```

Comment:
`LIBSODIUM_REPO`

Reply:
I added LIBSODIUM_REPO description into below

Reply:
I mean, there's a typo. You wrote `LIPSODIUM_REPO`

Reply:
Ah, got it)